### ISSUE TITLE
sessionstore unit test failing locally on MacOS

### ISSUE THREAD

**Issue by diracdeltas** [brave/browser-laptop#13907]
This unit test fails for me locally on the current master (a970195fe5d83783d572b6b22cc0e3514fa8ea7a):

```
sessionStore unit tests loadAppState when checking data.lastAppVersion when it does NOT match app.getVersion calls cleanAppData:

      AssertionError [ERR_ASSERTION]: promise was rejected: {}
      + expected - actual

      -false
      +true
```

**Comment by NejcZdovc** [comment]
this is failing from 0.22.x up

**Comment by diracdeltas** [comment]
i have a fix; it was caused by #13819
